Ds3/ResponseParsers/GetObjectResponseParser.cs

```diff
@@ -39,15 +39,14 @@ public GetObjectResponse Parse(GetObjectRequest request, IWebResponse response)
                     request.GetByteRanges().Any()
                         ? HttpStatusCode.PartialContent
                         : HttpStatusCode.OK
-                );
+                    );
                 using (var responseStream = response.GetResponseStream())
                 {
-                    responseStream.CopyTo(request.DestinationStream, this._copyBufferSize);
+                    StreamsUtil.BufferedCopyTo(responseStream, request.DestinationStream, _copyBufferSize);
                 }
-                return new GetObjectResponse(
-                    metadata: ResponseParseUtilities.ExtractCustomMetadata(response.Headers)
-                );
+                return new GetObjectResponse(ResponseParseUtilities.ExtractCustomMetadata(response.Headers)
+                    );
```

Comment:
formatting seems really funky here.  Can't tell what starts and ends where.  line 42 closes 37?  Line 48 closes... ???